interface/usergroup/usergroup_admin_add.php

```diff
@@ -237,7 +238,16 @@ function authorized_clicked() {
 <tr>
 <td style="width:150px;"><span class="text"><?php echo xlt('Username'); ?>: </span></td><td style="width:220px;"><input type="text" name="rumple" style="width:120px;" class="form-control"><span class="mandatory"></span></td>
 <?php if (empty($GLOBALS['gbl_ldap_enabled']) || empty($GLOBALS['gbl_ldap_exclusions'])) { ?>
-<td style="width:150px;"><span class="text"><?php echo xlt('Password'); ?>: </span></td><td style="width:250px;"><input type="password" style="width:120px;" name="stiltskin" class="form-control"><span class="mandatory"></span></td>
+<td style="width:150px;">
+    <span class="text"><?php echo xlt('Password'); ?>:</span>
+</td>
+<td style="width:150px;">
+    <input type="password" style="width:120px;" name="stiltskin" id="stiltskin" class="form-control" onkeyup="checkPasswordStrength(this);">
+    <span class="mandatory"></span>
+    <!-- Password Strength Meter -->
+    <div id="password_strength_meter" style="width:120px; height:20px; border:1px solid #ccc; margin-top: 5px;"></div>
```

Comment:
try not to use use embedded styling (looks likes will need the width:120px since used throughout in this script). instead use bootstrap classes for styling (see this scripts and other scripts for examples)

Reply:
I think he's trying to stay consistent with the styling from line 238.  Probably why they copied it in the divs up higher in the script.